src/DotNetCore.CAP.AzureServiceBus/Options/AzureServiceBusOptionsBase.cs

```diff
@@ -145,16 +129,5 @@ public Func<ServiceBusReceivedMessage, IServiceProvider, List<KeyValuePair<strin
     /// </summary>
     public List<KeyValuePair<string, string>>? SQLFilters { get; set; }
```

Comment:
To maintain parity, I believe we should also add this property to the custom consumer configuration, what do you think?